tests/test_e2e.py

```diff
@@ -3,17 +3,16 @@
 from threading import Thread
 import pytest
 from selenium import webdriver
-from dotenv import load_dotenv
+from selenium.webdriver.common.keys import Keys
+from dotenv import find_dotenv, load_dotenv
 import requests
 
 import app
 
-load_dotenv()
 TRELLO_BASE_URL = 'https://api.trello.com/1'
 TRELLO_API_KEY = os.environ.get('TRELLO_API_KEY')
 TRELLO_TOKEN = os.environ.get('TRELLO_TOKEN')
```

Comment:
Your test fails when run from command line. You are trying to read these environment variables **before** your test has had a chance to run `load_dotenv`